package taskHandle

func taskErrHandle(err error) error {
	return err
}
